### DISCUSSION TITLE
Multisimulation

### DISCUSSION THREAD

**Discussion by SebGue** [Eomys/pyleecan#302] [Q&A]
Hello,

the multisimulation tutorial says that it is possible to do a multi-simulation of multi-simulations.
Maybe I am missing something but I think that this is not possible?!

Best regards, Sebastian

**Comment by BonneelP** [comment]
Hello,

In the architecture we thought about this possibility but it is not implemented yet ;)
The idea is to add a second property like var_simu and in Simu.run we check this new property first (if define => multisimu of multi simu, else check var_simu else single simulation). The process should hopefully be the same as for a normal var_simu (it should be just a new layer), the only difference is that the simu list will have var_simu != None. This would enable to create Parameter sweep on variable speed simulation for instance. Do you plan to work on this feature ? Do you need help ? 

By the way, we are currently focusing on the new version of our software MANATEE (this is why we are a bit less active recently) but I should submit a PR this week with all the new slot/hole schematics and some GUI improvements ;) (https://github.com/EOMYS-Public/pyleecan/tree/Slot)
![image](https://user-images.githubusercontent.com/20533034/106026195-ce412180-60c9-11eb-93ee-95d899ab05a6.png)
![image](https://user-images.githubusercontent.com/20533034/106026163-c2edf600-60c9-11eb-9975-f3db23dab6fb.png)


Best regards,
Pierre

**Comment by SebGue** [comment]
Hello Pierre,

thanks for you helpful feedback. Actually I don't wanted to add such feature but I guess I may do it nevertheless.
I had a look at VarSimu and OptiProblem. I think they have strong similarities so I may also add a 'simu' or 'ref_simu' prop. to VarSimu that can contains the reference (multi-) simulation as you adviced.
I may also add the 'preprocessing' prop to VarSimu. So it will be possible to set all (dependend) parameters in one sweep after the actual parameter setters. This also relates to the question #296 (Feedback very welcome :-) ).

If you got futher ideas on this issue, let me know.

Best regards, Sebastian
